File: src/test/java/coreTDD/diamond/DiamondFactoryTest.java
package coreTDD.diamond;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class DiamondFactoryTest {

    private DiamondFactory diamondFactory = new DiamondFactory();

    @Test
    public void returns_diamond_for_1() {
        assertThat(diamondFactory.constructDiamond(1)).containsExactly("1");
    }
}

File: src/test/kotlin/kotlinCoreTDDTests/diamond/DiamondFactoryTest.kt
package kotlinCoreTDDTests.diamond

import kotlinCoreTDD.diamond.DiamondFactory
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class DiamondFactoryTest {

    @Test
    fun constructDiamond_for1() {
        assertThat(DiamondFactory().constructDiamond(1)).isEqualTo(arrayOf("1"))
    }
}